common/src/main/java/com/wynntils/mc/mixin/BossHealthOverlayMixin.java

```diff
@@ -31,46 +32,52 @@
                             target =
                                     "Lnet/minecraft/network/protocol/game/ClientboundBossEventPacket;dispatch(Lnet/minecraft/network/protocol/game/ClientboundBossEventPacket$Handler;)V"))
     private void updatePre(ClientboundBossEventPacket packet, ClientboundBossEventPacket.Handler handler) {
+        Map<UUID, LerpingBossEvent> _events = events;
+
         packet.dispatch(new ClientboundBossEventPacket.Handler() {
             public void add(
-                    UUID id,
-                    Component name,
+                    @NotNull UUID id,
+                    @NotNull Component name,
                     float progress,
-                    BossEvent.BossBarColor color,
-                    BossEvent.BossBarOverlay overlay,
+                    BossEvent.@NotNull BossBarColor color,
+                    BossEvent.@NotNull BossBarOverlay overlay,
                     boolean darkenScreen,
                     boolean playMusic,
                     boolean createWorldFog) {
-                events.put(
+                _events.put(
                         id,
                         new LerpingBossEvent(
                                 id, name, progress, color, overlay, darkenScreen, playMusic, createWorldFog));
             }
 
-            public void remove(UUID id) {
-                events.remove(id);
+            public void remove(@NotNull UUID id) {
+                _events.remove(id);
             }
 
-            public void updateProgress(UUID id, float progress) {
-                if (!events.containsKey(id)) return;
-                ((LerpingBossEvent) events.get(id)).setProgress(progress);
+            public void updateProgress(@NotNull UUID id, float progress) {
+                if (!_events.containsKey(id)) return;
+                (_events.get(id)).setProgress(progress);
```

Comment:
The extra parenthesis do nothing here, right? It's just a relic of the old, unnecessary cast, I think. (Same goes for the other case below.)

Reply:
Yeah, there were casts here, I removed them but left the parenthesis here by accident.